docs/x509.rst

```diff
@@ -718,6 +718,68 @@ X.509 Extensions
 
         :returns: A list of values extracted from the matched general names.
 
+.. class:: CertificatePolicies
+
+    .. versionadded:: 0.9
+
+    The certificate policies extension is a list of one or more
+    :class:`PolicyInformation` instances. The object is iterable to get every
+    instance.
```

Comment:
I think you can rewrite this as "The certificate policy extension is an iterable, containing one or more PolicyInformation instances."